README.md

```diff
@@ -369,7 +595,21 @@ device.on("getTempResponse", function(data) {
 
 <div id="route_onexception"><h3>onException(<i>callback</i>)</h3></div>
 
-The *onException()* method allows you to configure a route-level response handler for requests that encounter runtime errors. A route-level onException handler will override the global onException handler set by [*Rocky.onTimeout()*](#rocky_onexception) for requests made to the specified route. The callback method takes two parameters: a [Rocky.Context](#context) object and the exception. The callback method will be executed for all requests that encounter runtime errors and do not have a route-level [onException](route_onexception) handler. This method should (but is not required to) send a response code of 500.
+This method allows you to configure a route-level response handler for requests that encounter runtime errors. A route-level onException handler will override the global onException handler set by Rocky’s[*onTimeout()*](#rocky_onexception) method for requests made to the specified route.
```

Comment:
This link is confusing - it takes me to the global onException function not the onTimeout method